great_expectations/datasource/data_connector/__init__.py

```diff
@@ -15,3 +15,7 @@
 from .inferred_asset_s3_data_connector import InferredAssetS3DataConnector
 from .inferred_asset_sql_data_connector import InferredAssetSqlDataConnector
 from .runtime_data_connector import RuntimeDataConnector
+
+from .configured_asset_azure_data_connector import (  # isort:skip
+    ConfiguredAssetAzureDataConnector,
+)
```

Comment:
Having this placed at the top causes an import error. Package/import management is a weakpoint of mine but do you think this is worth looking into @fjork3 ?